data/applications.yml

```diff
@@ -659,6 +659,28 @@
   dashboard_url: false
   description: 'GOV.UK Licensing (formerly ELMS, Licence Application Tool, & Licensify)'
 
+# Coronavirus
+
+- github_repo_name: govuk-coronavirus-vulnerable-people-form
+  private_repo: false
+  type: Specialist apps
+  production_hosted_on: paas
+  team: "#govuk-vulnerable-people-tech"
+  sentry_url: false # TODO - can the reviewer let me know what this should be?
```

Comment:
It looks like [this is set in the pipeline config](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/e9e151499ae9a5a5b53dde615172cbf30299b4bd/concourse/tasks/deploy-to-govuk-paas.yml#L46). I don't seem to have access to concourse to see what it's actually configured to though. I expect there's some kind of 'settings' button [here](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-vulnerable-people-form) if you do.

Reply:
Hmmm, the DSN doesn't look like the other URLs in this doc - it's like `https://$user:$password@sentry.io/5170680`

I think this should be a link to the view in sentry.

Reply:
This seems to come up with something, but I don't think it's working properly, as it doesn't have a project name: https://sentry.io/organizations/govuk/issues/?project=5170680

Reply:
where do you see the project name ben? Where should it be?
I see coronavirus-vulnerable-people form from the left most of the the three top drop downs?

and coronavirus-business-voulenteer for the one we're doing today?
https://sentry.io/organizations/govuk/issues/?project=5172100

Reply:
Yeah, I don't see it (the dropdown is just blank for me). I know there's a strange thing with Sentry where you have to add the project 'manually' sometimes. Perhaps that's it.

I also don't see any coronavirus projects here: https://sentry.io/organizations/govuk/projects/

Reply:
```suggestion
  sentry_url: https://sentry.io/organizations/govuk/issues/?project=5170680
```
If this is just for reference then the above URL should be right,
otherwise if this is supposed to be the DNS might need removing here as that's a secret?